src/NHibernate/AdoNet/ConnectionManager.cs

```diff
@@ -40,6 +40,7 @@ public class ConnectionManager : ISerializable, IDeserializationCallback
 
 		private readonly ConnectionReleaseMode _connectionReleaseMode;
 		private readonly IInterceptor _interceptor;
+		[NonSerialized]
 		private readonly List<ISessionImplementor> _dependentSessions = new List<ISessionImplementor>();
```

Comment:
No actually needed since serialization interface is implemented, added for consistency.